examples/next/package.json

```diff
@@ -16,10 +17,12 @@
     "next-global-css": "^1.1.1",
     "react": "latest",
     "react-dom": "latest",
+    "rimraf": "^3.0.2",
```

Comment:
Do we need this in dependencies and devDependencies?

Reply:
Oops, that's a typo. Thanks for the catch!
```suggestion
```